pkg/rancher-desktop/main/tray.ts

```diff
@@ -11,15 +11,18 @@ import Electron from 'electron';
 import { State } from '@pkg/backend/k8s';
 import * as kubeconfig from '@pkg/backend/kubeconfig';
 import { Settings, load } from '@pkg/config/settings';
+import { getIpcMainProxy } from '@pkg/main/ipcMain';
 import mainEvents from '@pkg/main/mainEvents';
 import { checkConnectivity } from '@pkg/main/networking';
 import Logging from '@pkg/utils/logging';
 import paths from '@pkg/utils/paths';
 import { openMain, send } from '@pkg/window';
 import { openDashboard } from '@pkg/window/dashboard';
 import { openPreferences } from '@pkg/window/preferences';
+import { VMBackend } from '~/backend/backend';
```

Comment:
Please use `@pkg` instead of `~`. (At least for me, my IDE keeps using `~` for some reason…)

Reply:
Thanks, my editor keeps doing the same for auto imports.. I think we might be able to control the default in `tsconfig.json`